util.py

```diff
@@ -69,6 +69,8 @@ def runCmd2(command, with_stdout=False, with_stderr=False, inputtext=None):
         logger.log("running %s caused an exception: %s" % (command, ex))
         raise
 
+    out = out.decode() if isinstance(out, bytes) else out
```

Comment:
You can set `universal_newlines=True` in `Popen()` and the I/O streams will be strings rather than bytes.

It would also be worth adding a docstring or a comment describing this for future users of runCmd2().